install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -152,4 +152,6 @@ DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_CUSTOMER_NWSL';
 
 ALTER TABLE `PREFIX_cart_rule` ADD  `reduction_exclude_special` TINYINT(1) UNSIGNED NOT NULL DEFAULT  '0' AFTER  `reduction_percent`;
 ALTER TABLE `PREFIX_product` ADD state INT UNSIGNED NOT NULL DEFAULT '1';
-ALTER TABLE `PREFIX_product` ADD KEY state (state, date_upd);
\ No newline at end of file
+ALTER TABLE `PREFIX_product` ADD KEY state (state, date_upd);
+
+INSERT INTO  `PREFIX_configuration` (`id_configuration` ,`id_shop_group` ,`id_shop` ,`name` ,`value` ,`date_add` ,`date_upd`) VALUES (NULL , NULL , NULL ,  'PS_ORDER_RECALCULATE_SHIPPING',  '1',  '0000-00-00 00:00:00',  '0000-00-00 00:00:00');
```

Comment:
Example in PhpStorm:
![phpstormlf](https://cloud.githubusercontent.com/assets/6775736/17446875/bf67b238-5b4b-11e6-81f1-c266524764ce.png)